docs/en/operation/User-CLI.md

```diff
@@ -208,29 +208,20 @@ Status: CANCELED
 The `logLevel` command returns the current value of or updates the log level of a particular class
 on specific instances. Users are able to change Alluxio server-side log levels at runtime.
 
-The command follows the format `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`,
+The command follows the format `alluxio logLevel --logName=NAME [--target=<master|workers|job_master|job_workers|host:port>] [--level=LEVEL]`,
```

Comment:
The target is the leading master, leading job_master?
maybe we can include this information (only change the leading master/job_master) somewhere